src/chrome/__mocks__/chrome.js

```diff
@@ -230,6 +252,10 @@ export const mockChrome = (opts = {}) => {
 
   const getOpenTabs = () => openTabs;
 
+  const closeWindow = () => {
+    openWindows = [];
+  };
```

Comment:
Possible, but not  used that way in tests and I don't think it would be useful to add (I try to keep the mock to the minimum possible since the chrome API is rather huge).